src/cookbook/games/achievements-leaderboard.md

```diff
@@ -0,0 +1,352 @@
+    Center** page, you might want to set up a leaderboard and several
+    achievements, depending on your game. Take note of the IDs of the
+    leaderboards and achievements you create.
+
+### Android
+
+To enable *Play Games Services* on Android:
+
+1.  If you haven't already, 
+    go to [Google Play Console](https://play.google.com/console/) 
+    and register your game there.  
+    
+    ![Screenshot of the 'Create app' button in Google Play Console](/assets/images/docs/cookbook/google-play-create-app.png)
+
+2.  Still in Google Play Console, select *Play Games Services* → *Setup
+    and management* → *Configuration* from the navigation menu and
+    follow their instructions.
+
+    -   This takes a significant amount of time and patience. Among
+        other things, you'll need to set up an OAuth consent screen in
+        Google Cloud Console. If at any point you feel lost, consult the
+        official [Play Games Services guide](https://developers.google.com/games/services/console/enabling).    
+         
+        ![Screenshot showing the Games Services section in Google Play Console](/assets/images/docs/cookbook/play-console-play-games-services.png)
+
+3.  When done, you can start adding leaderboards and achievements in
+    **Play Games Services** → **Setup and management**. Create the exact
+    same set as you did on the iOS side. Make note of IDs.
+
+4.  Go to **Play Games Services → Setup and management → Publishing**.
+
+5.  Click **Publish**. Don't worry, this doesn't actually publish your
+    game. It only publishes the achievements and leaderboard. Once a
+    leaderboard, for example, is published this way, it cannot be
+    unpublished.
+
+6.  Go to **Play Games Services** **→ Setup and management →
+    Configuration → Credentials**.
+
+7.  Find the **Get resources** button. It returns an XML file with the
+    Play Games Services IDs.
+
+    ```xml
+    <!-- THIS IS JUST AN EXAMPLE -->
+    <?xml version="1.0" encoding="utf-8"?>
+    <resources>
+        <!--app_id-->
+        <string name="app_id" translatable="false">424242424242</string>
+        <!--package_name-->
+        <string name="package_name" translatable="false">dev.flutter.tictactoe</string>
+        <!--achievement First win-->
+        <string name="achievement_first_win" translatable="false">sOmEiDsTrInG</string>
+        <!--leaderboard Highest Score-->
+        <string name="leaderboard_highest_score" translatable="false">sOmEiDsTrInG</string>
+    </resources>
+    ```
+
+8.  Add a file at `android/app/src/main/res/values/games-ids.xml`
+    containing the XML you received in the previous step.
+
+## 2. Sign in to the game service
+
+Now that you have set up *Game Center* and *Play Games Services*, and
+have your achievement & leaderboard IDs ready, it's finally Dart time.
+
+1.  Add a dependency on the [`games_services` package](https://pub.dev/packages/games_services).
+
+    ```shell
+    flutter pub add games_services
+    ```
```

Comment:
To be consistent with elsewhere on the site:

````suggestion
    ```terminal
    $ flutter pub add games_services
    ```
````

Reply:
Fixed